src/main/webapp/scroll-container.js

```diff
@@ -72,7 +72,8 @@ export class ScrollContainer extends HTMLDivElement {
    * Otherwise, does nothing.
    *
    */
-  scrollToTopOfContainer(element, forceScroll = false) {
+  scrollToTopOfContainer(
+      element = this.lastChild.firstChild, forceScroll = false) {
```

Comment:
What is element here?

Reply:
the `element` within the container that should be scrolled to... maybe `childElement` would be better?  but it isn't a direct child either

Reply:
Why was this change made?  I don't see scroll being called anywhere without an element specified